src/main/java/com/amazonaws/services/neptune/propertygraph/schema/ExportSpecification.java

```diff
@@ -121,40 +121,7 @@ public ExportPropertyGraphTask createExportTask(GraphSchema graphSchema,
     }
 
     public MasterLabelSchemas createMasterLabelSchemas(Collection<FileSpecificLabelSchemas> fileSpecificLabelSchemasCollection) {
```

Comment:
This was essentially a static method to assist in constructing a `MasterLabelSchemas` from a `Collection<FileSpecificLabelSchemas>`. The only non-static context utilized in this method was passing `graphElementType` into the `MasterLabelSchemas` constructor. All of the static code to convert from `Collection<FileSpecificLabelSchemas>` to `Map<Label, MasterLabelSchema>` has been migrated into a new overloaded constructor for `MasterLabelSchemas`